Source/Calling/AVSCallMember.swift

```diff
@@ -117,23 +117,55 @@ public struct AVSClient: Hashable {
     public let userId: UUID
     public let clientId: String
 
+    public init(userId: UUID, clientId: String) {
+        self.userId = userId
+        self.clientId = clientId
+    }
+
+    init?(userClient: UserClient) {
+        guard
+            let userId = userClient.user?.remoteIdentifier,
+            let clientId = userClient.remoteIdentifier
+        else {
+            return nil
+        }
+
+        self.init(userId: userId, clientId: clientId)
+    }
+
 }
 
-extension AVSClient {
+extension AVSClient: Codable {
 
-    struct CodingData: Codable {
+    enum CodingKeys: String, CodingKey {
+        
+        case userid
+        case clientid
```

Comment:
```suggestion
        case userId = "userid"
        case clientId= "clientid"
```

This way you can define the name-mapping without implementing `init(from decoder: Decoder)` and `encode(to encoder: Encoder)`